Corzbank.Data/Entities/Card.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Corzbank.Data.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Corzbank.Data.Entities
+{
+    public class Card: KeyEntity
+    {
+        public string CardNumber { get; set; }
+        public string ExpirationDate { get; set; }
+        public string CvvCode { get; set; }
+        public CardTypeEnum CardType { get; set; }
+        public PaymentSystemEnum PaymentSystem { get; set; }
+        public decimal Balance { get; set; }
+        public bool IsActive { get; set; }
+        public string SecretWord { get; set; }
+
+        public User User { get; set; }
+        public List<TransferCard> TransferCards { get; set; }
+        public List<DepositCard> DepositCards { get; set; }
```

Comment:
add an empty line after each prop in all the places, easier to read

Reply:
ok, what to do if I started another feature, but want to fix this one

Reply:
you should fix that in current branch (feature/dependency-injection-core) and then merge into your new branch

BTW: branch name is not correct. feature/creation-db-entities is better

Reply:
Should I push new feature to new branch, and then fix this bugs?

Reply:
Of course, each feature requires new branch